tests/manage/z_cluster/upgrade/test_check_osd_mon_post_ocs_upgrade.py

```diff
@@ -0,0 +1,57 @@
+import logging
+
+import pytest
+
+from ocs_ci.framework.testlib import (
+    post_ocs_upgrade,
+    post_ocp_upgrade,
+)
+from ocs_ci.ocs.cluster import validate_osd_pdb, CephCluster
+from ocs_ci.ocs.ocp import get_ocs_version
+from ocs_ci.ocs.resources.pod import get_all_pods
+from ocs_ci.ocs import constants
+from ocs_ci.helpers.helpers import get_mon_pdb
+
+log = logging.getLogger(__name__)
+
+
+@post_ocp_upgrade
+@post_ocs_upgrade
+@pytest.mark.polarion_id("OCS-2524")
+def test_check_osd_pdb_post_upgrade():
+    """
+    Test to verify OSD PDBs
+    1. Post OCP and OCS successful upgrades check for OSD PDBs
+    2. Rook-ceph-drain-canary pods disappeared after upgrade of OCS version to ocs-operator.v4.6.2-233.ci
+
+    """
+    assert validate_osd_pdb(), "No OSD PDBs present in the cluster"
+
+    # Check for rook-ceph-canary pods
+    ocs_version = get_ocs_version()
+    assert get_all_pods(
+        selector=constants.ROOK_CEPH_DRAIN_CANARY
+    ), f"Canary pods not found on {ocs_version}"
+
+
+@post_ocs_upgrade
+@pytest.mark.polarion_id("OCS-2449")
+def test_check_mon_pdb_post_upgrade():
+    """
+    Testcase to check disruptions_allowed and minimum
+    available mon count
+
+    """
+    ceph_obj = CephCluster()
+
+    # Check for mon count
+    mons_after_upgrade = ceph_obj.get_mons_from_cluster()
+    log.info(f"Mons after upgrade {mons_after_upgrade}")
+
+    disruptions_allowed, min_available_mon = get_mon_pdb()
+    log.info(f"Number of Mons Disruptions_allowed {disruptions_allowed}")
+    log.info(f"Minimum_available mon count {min_available_mon}")
+
+    # The PDB values are considered from OCS 4.5 onwards.
+    assert disruptions_allowed == 1, "Mon Disruptions_allowed count not matching"
+    assert min_available_mon == 2, "Minimum available mon count is not matching"
```

Comment:
Are we purposefully removing these lines from test https://github.com/red-hat-storage/ocs-ci/blob/master/tests/manage/z_cluster/upgrade/test_check_mon_pdb_post_upgrade.py#L38-L51 ?